<-PYT:import moduletomock
<-PYT:moduletomock.__name__
->RET:'moduletomock'
<-PYT:moduletomock.__path__
->RET:raise AttributeError("module 'moduletomock' has no attribute '__path__'")
<-PYT:import no_such_module
->RET:raise ModuleNotFoundError("No module named 'no_such_module'")
<-PYT:moduletomock.no_such_method
->RET:raise AttributeError("module 'moduletomock' doesn't REALLY have an attribute 'no_such_method', stupid!")
<-PYT:moduletomock.call_function('argument', faking=True)
->RET:raise TypeError("call_function() got an unexpected keyword argument 'faking'")
<-PYT:import fakeurllib
<-PYT:fakeurllib.__name__
->RET:'fakeurllib'
<-PYT:fakeurllib.__path__
->RET:raise AttributeError("module 'fakeurllib' has no attribute '__path__'")
<-PYT:fakeurllib.urlopen('foo://no.such.site')
->RET:raise Instance('URLError(OSError)', 'urlerror_foo___no_such_site')
<-PYT:urlerror_foo___no_such_site.args
->RET:('No such site, you wally!',)
<-PYT:fakeurllib.urlopen('foo://another.no.such.site')
->RET:raise Instance('URLError', 'urlerror1')
<-PYT:urlerror1.__str__()
->RET:"<urlopen error (-2, 'Now you're really getting on my nerves!')>"
<-PYT:urlerror1.__suppress_context__
->RET:False
<-PYT:urlerror1.__bool__
->RET:raise AttributeError("'URLError' object has no attribute '__bool__'")
<-PYT:urlerror1.__len__
->RET:raise AttributeError("'URLError' object has no attribute '__len__'")
